.github/workflows/build.yml

```diff
@@ -48,4 +48,6 @@ jobs:
         uses: tauri-apps/tauri-action@v0
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          AW_WEBUI_DIR: ${{ github.workspace }}/aw-webui/dist
```

Comment:
This should be set in the Makefile or something, not in the CI config.

Reply:
The action does not call the ```make``` command. Unless we modify the action to do that.